accessibility/css/reveal.scss

```diff
@@ -0,0 +1,1319 @@
+/*!
```

Comment:
Hi there. I made changes to the reveal CSS to make the presentation more accessible. I will check where I made them and we can delete once I verify if that is helpful.

> On Oct 4, 2015, at 11:38 AM, Brenda Jin <notifications@github.com> wrote:
> 
> In accessibility/css/reveal.scss:
> 
> > @@ -0,0 +1,1319 @@
> > +/*!
> I don't think we'll need these files if we already have a compiled reveal.css file, unless you've made modifications?
> 
> —
> Reply to this email directly or view it on GitHub.
>